### PR TITLE
docs: added README

### PR DESCRIPTION
**Pull request by Souheil-Yazji**
Let me know if anything else should be added to the README.

#1

### CODE REVIEW

**Review by ChristopheTran — COMMENTED** [review]
Looks good. Just a thought but should we include our group number?

**Comment by Souheil-Yazji** [comment]
> Looks good. Just a thought but should we include our group number?

done!

File: README.md [hunk 1]
```diff
@@ -1 +1,15 @@
...
+## Authors
+Group 4
+- Michael Fan
+- Derek Shao
+- Hoang Bui
+- Christophe Tran
+- Souheil Yazji
+
+## Contributions
+Use the following link for [commit convetion](https://www.conventionalcommits.org/en/v1.0.0/). All contributions to the project should pretain to an [Issue](https://github.com/mpfan/SYSC-3303-Elevator-Simulator/issues). Individual contributions to the project should be made through Pull Requests into the Dev branch.
```
Comment (dershao) [inline README.md]: Commit Convention vs Commit convetion

**Review by dershao — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Souheil-Yazji) [reply README.md]: Done!

**Review by dershao — APPROVED** [review]

**Pull request merged**